static/js/components/material/Button.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+import React from 'react';
+
+export default class Button extends React.Component {
+  render() {
+    const { children, className, ...otherProps } = this.props;
+
+    return <button
+      className={className ? `mdc-button ${className}` : 'mdc-button'}
```

Comment:
I'm not too particular, it just reads easier to me, but that's highly personal preference. I do think we could be consistent though. I'm not sure how well this pattern works in situations with multiple conditional classes, but that can be worked around by concatenating together a single `className`.

To throw a third alternative out there, something like [classnames](https://github.com/JedWatson/classnames) might be a better approach if we end up with more complicated `className` cases